templates/celery.j2

```diff
@@ -0,0 +1,11 @@
+[program:celery]
+user={{ www_user }}
+directory={{ openwisp2_path }}
+command={{ openwisp2_path }}/env/bin/celery -A openwisp2 worker -l info --pool=eventlet --concurrency=30 -Ofair
```

Comment:
let's make `--concurrency` configurable with a default conservative value of `10`, we can use variable name of `openwisp2_eventlet_concurrency`.